recipes-nodes/rbuilder-bidding/rbuilder-bidding.bb

```diff
@@ -3,7 +3,6 @@ LICENSE = "CLOSED"
 FILESEXTRAPATHS:prepend := "${THISDIR}:"
 
 SRC_URI += "file://init \
-            file://bidding-service.toml \
```

Comment:
this never existed the first place, right?

Reply:
yes, a leftover from the local development